pkg/client/client.go

```diff
@@ -182,11 +182,19 @@ func (c *client) RESTMapper() meta.RESTMapper {
 func (c *client) Create(ctx context.Context, obj Object, opts ...CreateOption) error {
 	switch obj.(type) {
 	case *unstructured.Unstructured:
-		return c.unstructuredClient.Create(ctx, obj, opts...)
+		err := c.unstructuredClient.Create(ctx, obj, opts...)
```

Comment:
How would that error ever occur here? The whole point of `unstructured` is to not need types.